lms/djangoapps/ccx/utils.py

```diff
@@ -34,6 +35,29 @@
 log = logging.getLogger("edx.ccx")
 
 
+def get_ccx_creation_dict(course):
+    """
+    Return dict of rendering create ccx form.
+
+    Arguments:
+        course (CourseDescriptorWithMixins): An edx course
+
+    Returns:
+        dict: A attribute dict for view rendering
+    """
+    # Translators: ccx_connector is url for CCX Connector application for managing creation of CCXs.s
```

Comment:
I think this comment needs to be directly above the line with the translated string.  You can check the extracted .po files to see if the translators comment is there.  But this comment doesn't seem to relate to the translated string anyway? Do you need it?